src/components/clusterConfiguration/BasicNetworkFields.tsx

```diff
@@ -1,20 +1,37 @@
 import React from 'react';
 import { HostSubnets, ClusterConfigurationValues } from '../../types/clusters';
 import { InputField, SelectField } from '../ui/formik';
-import { useFormikContext } from 'formik';
+import { useFormikContext, useField } from 'formik';
 
 type BasicNetworkFieldsProps = {
   hostSubnets: HostSubnets;
 };
 
 const BasicNetworkFields: React.FC<BasicNetworkFieldsProps> = ({ hostSubnets }) => {
   const { validateField } = useFormikContext<ClusterConfigurationValues>();
+  const [clusterName] = useField({ name: 'name' });
+  const [baseDnsField] = useField({ name: 'baseDnsDomain' });
+  const [hostSubnet] = useField({ name: 'hostSubnet' });
```

Comment:
useField is used to construct custom input fields. the values are available through useFormikContext